rfd/0174-kube-rbac-bootstrap.md

```diff
@@ -59,45 +112,22 @@ spec:
   allow:
     kubernetes_labels:
       'env': 'staging'
-    kubernetes_resources:
-      - kind: pod
-        namespace: "main-company-app"
-        name: "*"
-    kubernetes_groups:
-    - default-view
-    kubernetes_users:
-    - developer
+    kubernetes_permissions:
+      - namespaces: ["main-company-app"]
+      - resources: ["pod"]
+        apiGroups: [""]
+        verbs: ["get", "list", "watch"]
   deny: {}
 ```
 
 After that Alice assigns this role to the developers she wants to try out and at this point she is done. Alice will not need to create a 
-separate role binding in Kubernetes for it to work - once the cluster is enrolled in Teleport, the default-view group binding is already there.
+separate role binding in Kubernetes for it to work - once the role is created Teleport will automatically create required Role/Binding in the
+staging Kubernetes cluster.
 
 #### Upgrade path
```

Comment:
I've mentioned in the RFD that we'll use `system:master` (or `cluster-admin`) group for impersonation to creating role/bindings, to avoid the need to update the chart and our permissions. I think it should be sufficient.

Reply:
Oh, my bad I missed this 😅